backend/compound_library/src/main.rs

```diff
@@ -132,7 +132,8 @@ async fn main() {
         }
         Cli::Schema(args) => {
             let schema = root_schema_builder().finish();
-            let schema_string = schema.sdl();
+            let export_option = async_graphql::SDLExportOptions::default().federation();
+            let schema_string = schema.sdl_with_options(export_option);
```

Comment:
Don't think we need to be so verbose about this :stuck_out_tongue: 

```suggestion
            let schema_string = schema.sdl_with_options(SDLExportOptions::default().federation());
```